src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -176,7 +176,7 @@ await _engine.WaitForState(RachisConsensus.State.Leader)
                             }
                             foreach (var node in nodesChanges.addedValues)
                             {
-                                var task = ClusterMaintenanceSupervisor.AddToCluster(node.Key, clusterTopology.GetUrlFromTag(node.Key)).ContinueWith(t =>
+                                var task = ClusterMaintenanceSupervisor.AddToCluster(node.Key, clusterTopology.GetUrlFromTag(node.Key), clusterTopology.ApiKey).ContinueWith(t =>
```

Comment:
`clusterTopology.ApiKey` if this is for internal use only, and IIRC it is so, we can drop it entirely.